modules/tests/test/dataobject/operations.go

```diff
@@ -32,9 +42,29 @@ func WaitForSuccessfulDataSource(cdiClientSet cdicliv1beta1.CdiV1beta1Interface,
 	})
 }
 
-func IsDataVolumeImportSuccessful(cdiClientSet cdicliv1beta1.CdiV1beta1Interface, namespace string, name string) bool {
+func WaitForBoundPVC(k8sClient *kubernetes.Clientset, namespace, name string, timeout time.Duration) error {
+	return wait.PollImmediate(constants.PollInterval, timeout, func() (bool, error) {
+		pvc, err := k8sClient.CoreV1().PersistentVolumeClaims(namespace).Get(context.TODO(), name, metav1.GetOptions{})
+		if err != nil {
+			return true, err
+		}
+		return isPVCStatusBound(pvc), nil
+	})
+}
+
+func IsDataVolumeImportSuccessful(k8sClient *kubernetes.Clientset, cdiClientSet cdicliv1beta1.CdiV1beta1Interface, namespace string, name string) bool {
```

Comment:
Can't the `kubecli.KubevirtClient` be used for everything? It implements `kubernetes.Clientset` and provides the CDI client with method `CdiClient()`.